paf-mvp-audit/src/view.ts

```diff
@@ -118,7 +118,7 @@ export class View implements IView {
     this.auditContainer.classList.add('ok-ui');
 
     // If the pop up is valid then append the container and store a reference to the pop up element.
-    this.root = this.outerContainer.attachShadow({ mode: 'closed' });
+    this.root = this.outerContainer.attachShadow({ mode: 'open' });
```

Comment:
There doesn't seem to be a benefit to use `closed` mode, and it's preventing any kind of testing with Cypress.
See https://blog.revillweb.com/open-vs-closed-shadow-dom-9f3d7427d1af